src/main/java/com/microsoft/sqlserver/jdbc/SQLServerXAResource.java

```diff
@@ -765,15 +765,8 @@ private XAReturnValue dtc_XA_interface(int nType, Xid xid, int xaFlags) throws X
             if (xaLogger.isLoggable(Level.FINER))
                 xaLogger.finer(toString() + " exception:" + ex);
 
-            if (ex.getMessage().equals(SQLServerException.getErrString("R_noServerResponse"))
-                    || TransientError.isTransientError(ex.getSQLServerError()) || isResourceManagerFailure(ex)) {
-                XAException e = new XAException(ex.toString());
-                e.errorCode = XAException.XAER_RMFAIL;
-                throw e;
-            }
-
             XAException e = new XAException(ex.toString());
-            e.errorCode = XAException.XAER_RMERR;
+            e.errorCode = XAException.XAER_RMFAIL;
```

Comment:
Remove unused import `import com.microsoft.sqlserver.jdbc.SQLServerError.TransientError;` (Added in original XA PR [#2078].

Also can remove `import java.net.SocketException;` (Added in 2nd XA PR [#2132]